pureStorageDriver/README.md

```diff
@@ -60,31 +26,78 @@ More details on using Read-Write-Many (RWX) volumes with Kubernetes can be found
 
 Whilst there are some default `StorageClass` definitions provided by the PSO installation, refer [here](../docs/custom-storageclasses.md) for more details on these default storage classes and how to create your own custom storage classes that can be used by PSO.
 
-## How to install
+## Installation
+
+### Configure NTP
+
+PSO CSI driver requires all compute node clocks to be within 500ms.
+
+Ensure that an implementation of NTP is installaed and running on all cluster members, even those running as virtual machines.
+
+Example implementations include `ntp`, `chronyd`, `kvm-clock` and `system-timed`
+
+### Install the plugin in a separate namespace (i.e. project)
 
-Add the Pure Storage helm repo
+For security reason, it is strongly recommended to install the plugin in a separate namespace/project. **Do not use the `default` namespace.**
+
+Make sure the namespace exists, otherwise create it before installing the plugin.
+
+```bash
+kubectl create namespace <pso-namespace>
+```
+
+### Configure Helm
+
+Add the Pure Storage PSO helm repository to your helm installation.
 
 ```bash
 helm repo add pure https://purestorage.github.io/pso-csi
 helm repo update
 helm search repo pureStorageDriver -l
```

Comment:
don't need to mention `--devel` flag here for any rc releases? they are not listed in the result without this flag.

Reply:
Deliberately removed those. 
We shouldn't be advertising them to the world.
If customers are requested to use devel versions we can tell them how to do it - IMHO